pkg/unshare/unshare_linux.go

```diff
@@ -464,11 +464,6 @@ func bailOnError(err error, format string, a ...interface{}) { // nolint: golint
 
 // MaybeReexecUsingUserNamespace re-exec the process in a new namespace
 func MaybeReexecUsingUserNamespace(evenForRoot bool) {
-	// If we've already been through this once, no need to try again.
```

Comment:
Doesn't this change it from Maybe to just Reexec?

Reply:
we don't always reexec unless `evenForRoot` is set.

If `evenForRoot` is not set, then we check if CAP_SYS_ADMIN is in the effective capabilities

Reply:
> we don't always reexec unless `evenForRoot` is set.

Uh? The only codepath in this function that doesn't reexec doesn't involve `evenForRoot` in any way.